cmd/clusterctl/pkg/client/cluster/move_test.go

```diff
@@ -0,0 +1,636 @@
+/*
+Copyright 2020 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cluster
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+	"testing"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/cluster-api/cmd/clusterctl/pkg/internal/test"
+)
+
+func TestObjectGraph_getDiscoveryTypeMetaList(t *testing.T) {
+	type fields struct {
+		proxy Proxy
+	}
+	tests := []struct {
+		name    string
+		fields  fields
+		want    []metav1.TypeMeta
+		wantErr bool
+	}{
+		{
+			name: "Return CRDs + ConfigMap & Secrets",
+			fields: fields{
+				proxy: test.NewFakeProxy().
+					WithObjs(
+						test.FakeCustomResourceDefinition("foo", "Bar", "v1", "v2"),
```

Comment:
Missing this function definition

Reply:
[1] waiting for #2059 to merge (I broke up in two PRs to keep this one as focused as possible)